.github/workflows/pull_request.yml

```diff
@@ -1,7 +1,17 @@
 
 name: Build Chain
 
-on: [pull_request]
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths-ignore:
+      - 'LICENSE*'
+      - '.gitignore'
+      - '*.md'
+      - '*.adoc'
+      - '*.txt'
+      - 'build/**'
+      - 'ide-configuration/**'
```

Comment:
There's no currently an `ide-configuration`. Please check whether it's necessary. Otherwise LGTM.

Reply:
`ide-configuration` is no longer necessary on the `main` branch, as it was moved into the `build/` directory.

Reply:
```suggestion
```